packages/lib/src/toolbar/controls/Histogram/Indicators.tsx

```diff
@@ -0,0 +1,52 @@
+import type { Domain } from '@h5web/shared';
+import { Drag } from '@visx/drag';
+
+import type { Size } from '../../../vis/models';
+import styles from './Histogram.module.css';
+import Indicator from './Indicator';
+
+interface Props {
+  positions: Domain;
+  size: Size;
+  onChange?: (domain: Domain) => void;
+}
+
+function Indicators(props: Props) {
+  const { positions, size, onChange } = props;
+  const { height, width } = size;
+  const [xMin, xMax] = positions;
+
+  return (
+    <g className={styles.indicators}>
+      <Drag
+        height={height}
+        width={width}
+        x={xMin}
+        onDragEnd={({ dx }) => onChange?.([xMin + dx, xMax])}
+        restrict={{ xMin: 0, xMax }}
+      >
+        {(dragState) => (
+          <Indicator x={xMin} interactive={!!onChange} {...dragState} />
```

Comment:
Instead of having a separate `interactive` prop, could you maybe just spread `dragState` conditionally?

```tsx
{...onChange && dragState}
```

Then in `Indicator`, you can have:

```tsx
  const handlers = useMemo(
    () => ({
            onPointerMove: dragMove,
            onPointerUp: dragEnd && (e: PointerEvent<Element>) => {
              (e.target as Element).releasePointerCapture(e.pointerId);
              dragEnd(e);
            },
            onPointerDown: dragStart && (e: PointerEvent<Element>) => {
              (e.target as Element).setPointerCapture(e.pointerId);
              dragStart(e);
            }
          }),
    [dragEnd, dragMove, dragStart, interactive]
  );
```

You can add the cursor style conditionally in the JSX based on `dragStart` or whatever.